packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -1596,17 +1630,19 @@ contract Deploy is Deployer {
                             _params.weth,
                             IAnchorStateRegistry(mustGetAddress("AnchorStateRegistryProxy")),
                             cfg.l2ChainID()
-                        )
-                    )
+                        ),
```

Comment:
Are you on the version of forge defined in `versions.json`? Looks like there are some listing things that are off

Reply:
Yeah i am,. Is the issue still present?